File: comms/bulk-messaging/senders/resolve-sender/resolve-sender.php
<?php

namespace Example;

use Twilio\Comms\TwilioComms;
use Twilio\Comms\Senders\Types\SendersResolveRequestSenderPoolId;
use Twilio\Comms\Types\CommunicationRecipient;
use Twilio\Comms\Types\CommunicationChannel;

$client = new TwilioComms(
    accountId: 'TWILIO_ACCOUNT_SID',
    authToken: 'TWILIO_AUTH_TOKEN',
);
$client->senders->resolve(
    new SendersResolveRequestSenderPoolId([
        'recipientAddresses' => [
            new CommunicationRecipient([
                'address' => '+14153902337',
                'channel' => CommunicationChannel::Phone->value,
            ]),
            new CommunicationRecipient([
                'address' => '+14153902337',
                'channel' => CommunicationChannel::Whatsapp->value,
            ]),
            new CommunicationRecipient([
                'address' => 'davidpletnjov@example.com',
                'channel' => CommunicationChannel::Email->value,
            ]),
        ],
    ]),
);

File: comms/bulk-messaging/senders/resolve-sender/resolve-sender.java
package com.example.usage;

import com.twilio.sdk.TwilioComms;
import com.twilio.sdk.resources.senders.types.SendersResolveRequest;
import com.twilio.sdk.resources.senders.types.SendersResolveRequestSenderPoolId;
import com.twilio.sdk.types.CommunicationChannel;
import com.twilio.sdk.types.CommunicationRecipient;
import java.util.Arrays;

public class Example {
    public static void main(String[] args) {
        TwilioComms client = TwilioComms
            .builder()
            .credentials("TWILIO_ACCOUNT_SID", "TWILIO_AUTH_TOKEN")
            .build();

        client.senders().resolve(
            SendersResolveRequest.of(
                SendersResolveRequestSenderPoolId
                    .builder()
                    .recipientAddresses(
                        Arrays.asList(
                            CommunicationRecipient
                                .builder()
                                .address("+14153902337")
                                .channel(CommunicationChannel.PHONE)
                                .build(),
                            CommunicationRecipient
                                .builder()
                                .address("+14153902337")
                                .channel(CommunicationChannel.WHATSAPP)
                                .build(),
                            CommunicationRecipient
                                .builder()
                                .address("davidpletnjov@example.com")
                                .channel(CommunicationChannel.EMAIL)
                                .build()
                        )
                    )
                    .build()
            )
        );
    }
}